webapp/channels/src/components/admin_console/localization/localization.scss

```
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+.autotranslation-section-header {
+    display: flex;
+    width: 100%;
+    min-width: 0; // prevents overflow in flex layouts
+    align-items: flex-start;
+    justify-content: space-between;
+    gap: 24px;
+}
+
+.autotranslation-section-toggle {
+    display: flex;
+    align-items: center;
+    align-self: center;
+}
+
+.localization-section-title {
+    margin: unset;
+    font-size: 16px;
+}
+
+.localization-section-description, .helpText {
```

```suggestion
.localization-section-description {
```

Do we need `.help-text` here? Seems unneccessary and might affect other parts of the app if we add this rule in this scope. I think we're safe to remove.